gsonpath-sample/src/main/java/gsonpath/interface_test/InterfaceExample.java

```diff
@@ -19,4 +19,25 @@
     Double[] getDoubleArrayExample();
 
     Boolean[] getBooleanArrayExample();
+
+    static InterfaceExample create(Integer intExample,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Method has 8 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00107)